packages/ckeditor5-table/src/tablelayout/tablelayoutediting.ts

```diff
@@ -204,13 +249,9 @@ function isTableTypeContent( viewTable: ViewElement ): boolean {
 }
 
 /**
- * Returns callback that checks if the element is a layout table.
+ * Checks if the element is a layout table.
  * It is used to disallow attributes or children that is managed by `Schema`.
  */
-function layoutTableCheck() {
-	return ( context: SchemaContext ) => {
-		if ( context.endsWith( 'table' ) && context.last.getAttribute( 'tableType' ) == 'layout' ) {
-			return false;
-		}
-	};
+function layoutTableCheck( context: SchemaContext ) {
+	return !( context.endsWith( 'table' ) && context.last.getAttribute( 'tableType' ) == 'layout' );
```

Comment:
I'm not sure if returning `true` would not allow on every other element. Please check.